controllers/dependencies.go

```diff
@@ -2,16 +2,16 @@ package controllers
 
 import (
 	"context"
-
+	"fmt"
 	"os"
-
 	"path/filepath"
 
-	git "github.com/go-git/go-git/v5"
```

Comment:
Nice catch, seems refactor import dropped that.

Reply:
Added back alias.

Reply:
Looks like it happened again. Bad squash?

Reply:
It seems to work. Must be an alias for it somewhere else?

Reply:
GoLand optimizes imports, the alias for this is still git, so it's technically unused and gets removed. But there is another package that is in `go.pkg` that also has `git` as an alias so we probably want to keep the alias to avoid collisions. To avoid both issues (collision and the alias getting dropped), I called this `gitv5` the `go.pkg` one is v4. LMK your thoughts.